packages/rpc-core/src/rpc-subscriptions/slots-updates-notifications.ts

```diff
@@ -0,0 +1,20 @@
+import { Slot } from '../rpc-methods/common';
+import { UnixTimestamp } from '../unix-timestamp';
+
+type SlotsUpdatesNotificationsApiNotification = Readonly<{
+    // Only present for 'createdBank' and notifications
+    parent?: Slot;
+    slot: Slot;
+    timestamp: UnixTimestamp;
+    type: 'completed' | 'createdBank' | 'dead' | 'firstShredReceived' | 'frozen' | 'optimisticConfirmation' | 'root';
+}>;
+
+export interface SlotsUpdatesNotificationsApi {
+    /**
+     * Subscribe to receive a notification from the validator on a variety of updates on every slot
+     */
+    slotsUpdatesNotifications(
```

Comment:
Oof. This really underscores how grammatically incorrect this method name is. It should have been `slotUpdateNotification`/`slotUpdateSubscribe`.